presto-main/src/test/java/io/prestosql/operator/scalar/TestVarbinaryFunctions.java

```diff
@@ -465,26 +466,21 @@ private static String encodeHex(String value)
         return base16().encode(value.getBytes(UTF_8));
     }
 
-    private static SqlVarbinary sqlVarbinary(String value)
+    public static SqlVarbinary sqlVarbinary(String value)
```

Comment:
UTF_8 is implicit and not obvious.
maybe limit this to all-ASCII String values?